locustfile.py

```diff
@@ -0,0 +1,73 @@
+"""
+Load testing script.
+
+Run via:
+
+$ locust
+
+Then visit http://localhost:8089 on host to run a test via the web UI.
+"""
+from locust import HttpLocust, TaskSet, task
+from hashlib import sha1
+from random import randint
+
+
+# NOTE: Change this based on your test user & plan
+USERNAME = 'testuser'
+PASSWORD = 'Test123$'
+PLAN_ID = 5
+
+# Configuration/instance specific. Assumes PA instance.
+NUM_DISTRICTS_IN_PLAN = 18
+REASSIGNMENT_POST_DATA = {
+    'geolevel': 2,
+    'geounits': 731,
+    'version': 0,
+}
+
+
+class UserActions(TaskSet):
+
+    def on_start(self):
+        self.login()
+
+    def login(self):
+        response = self.client.get('/accounts/login/')
+        csrftoken = response.cookies['csrftoken']
+        # Connection is unencrypted so the password is hashed before being sent
+        # to backend.
+        hashed_password = sha1(PASSWORD.encode('utf-8')).hexdigest()
+        self.client.post('/accounts/login/',
```

Comment:
It'd be worth a comment to note that this sets a cookie on the `self.client` object.